config/irsl/irsl_assembler_config.yaml

```
@@ -3623,6 +3623,1318 @@ PartsSettings:
       types : [PC1]
       translation: [48.0, 8.0, 0.0]
       rotation: [1., 0., 0., 180.0]
+  -
+    type: vehicle-base-ps
+    class: Base plate (Plag and socket)
```

@Hiroaki-Masuzawa
こちらのPlugのスペルの修正もお願いできますでしょうか。🙇

```suggestion
    class: Base plate (Plug and socket)
```

---

先ほど修正しました．